public/static/app/perfdiscovery/templates.js

```diff
@@ -35,7 +35,11 @@ mciModule.run(function($templateCache) {
   $templateCache.put('perf-discovery-bfs',
     '<div class="ui-grid-cell-contents">' +
       '<span ng-repeat="item in COL_FIELD">' +
-        '<span ng-bind-html="item.key | jiraLinkify : grid.appScope.$ctrl.jiraHost"></span>' +
+        '<span ' +
+          'ng-class="!item.isOpen && \'strikethrough\'" ' +
```

Comment:
I think we should only strikethrough if the BF is not one of the OPEN_STATUSES

Reply:
`isOpen` is virtual field, which is equal to `true` when BF status is in `OPEN_STATUSES`. I have to do this operation on one of the previous steps, and I decided to store the result into 'virtual' field. I can prefix it with `_` to make it more obvious that the field is 'internal' or 'virtual'